### DISCUSSION TITLE
heya!

### DISCUSSION THREAD

**Discussion by JipBoesenkool** [tonytins/mysimulation#5] [General]
Hey!
I'm interested in joining the project is there a way we could connect so you can talk about your ideas and where you want the project to go?

**Comment by tonytins** [comment]
Of course! I'll set up a Discord server. While I wasn't expecting anything to come out of this in such a short time, your help and enthusiasm is most welcome.

**Reply by JipBoesenkool** [reply]
awesome! Right place right time I suppose, just started looking into TSO this week!
